docs/application/native/getting-started/mobile/first-app.md

```diff
@@ -1,20 +1,14 @@
-
 # Creating Your First Tizen Mobile Native Application
 
 **Welcome to Tizen mobile native application development!**
 
 A mobile native application is created using the C language, and can be run on Tizen mobile devices. The application uses the native API, which provides various interfaces to the device hardware allowing you to take advantage of numerous capabilities tailored to run with limited device resources.
 
-Study the following instructions to help familiarize yourself with the Tizen [native application development
-process](../process/app-dev-process-n.md) as well as using the Tizen
-Studio and installing the created application on the emulator or target device. With the instructions, you can create and run a basic mobile native application, which displays some text on the screen with no user interaction:
+Study the following instructions to help familiarize yourself with the Tizen [native application development process](../../tutorials/process/app-dev-process-n.md) as well as using the Tizen Studio and installing the created application on the emulator or target device. With the instructions, you can create and run a basic mobile native application, which displays some text on the screen with no user interaction:
```

Comment:
Don't we need to change this `app-dev-process-n.md` into `app-dev-process.md`?

Reply:
Done.
However, I will update all files under getting-started directory next time. Please review this PR except the files under getting-started.